.github/workflows/python-tests.yml

```diff
@@ -0,0 +1,67 @@
+name: Python Tests
+on: [push, pull_request]
+
+jobs:
+  run-babelfish-python-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        id: checkout
+
+      - name: Install Dependencies
+        id: install-dependencies
+        if: always()
+        uses: ./.github/composite-actions/install-dependencies
+
+      - name: Build Modified Postgres
+        id: build-modified-postgres
+        if: always() && steps.install-dependencies.outcome == 'success'
+        uses: ./.github/composite-actions/build-modified-postgres
+      
+      - name: Compile ANTLR
+        id: compile-antlr
+        if: always() && steps.build-modified-postgres.outcome == 'success'
+        uses: ./.github/composite-actions/compile-antlr
+      
+      - name: Build Extensions
+        id: build-extensions
+        if: always() && steps.compile-antlr.outcome == 'success'
+        uses: ./.github/composite-actions/build-extensions
+      
+      - name: Install Extensions
+        id: install-extensions
+        if: always() && steps.build-extensions.outcome == 'success'
+        uses: ./.github/composite-actions/install-extensions
+
+      - name: Install Python
+        id: install-python
+        if: always() && steps.install-extensions.outcome == 'success'
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      
+      - name: Configure Python Environment
+        id: configure-python-environment
+        if: always() && steps.install-python.outcome == 'success'
+        run: |
+          cd ~
+          curl https://packages.microsoft.com/config/ubuntu/20.04/prod.list | sudo tee /etc/apt/sources.list.d/mssql-release.list
```

Comment:
This might not be necessary since we are grabbing prod.list in the install-dependencies workflow, but I'm not sure

Reply:
This ones target is different than that one and is python specific, not sure if its worth making this a common step.